components/compatimetrics_plots.py

```diff
@@ -85,11 +92,17 @@ def rmsd_matrix(predictions):
                 matrix[i, j] = round(root_mean_squared_difference(predictions[models[i]],
                                                                   predictions[models[j]]))
     matrix = pd.DataFrame(matrix, index=models, columns=models)
-    fig = px.imshow(matrix, text_auto=True, color_continuous_scale=[[0, 'lightblue'], [0.5, 'blue'], [1, 'purple']])
+    fig = px.imshow(matrix, color_continuous_scale=[[0, 'lightblue'], [0.5, 'blue'], [1, 'purple']],
+                    labels=dict(x="Model 1",
```

Comment:
In app I can't see this x-axis legend. It is only on the y-axis it is model 2. Fix it or remove the legend from the y-axis.

Reply:
Fixed